scripts/plugins/deploy_plugins.py

```diff
@@ -270,28 +270,79 @@ def Deploy_Plugins():
                 )
 
                 try:
-                    result = subprocess.run(
+                    # Tried this approach, and the output was not live output
+                    #  tested by adding the following in mounted plugin code
+                    #      sleep 3; echo "hello"; sleep 3; echo "hello 2";
+                    #  expected to see "hello" and then a pause, and then "hello 2"
+                    #  result: no output was shown until after the plugin deploy script finished
+                    # process = subprocess.Popen(
+                    #     [
+                    #         plugin_deploy_language,
+                    #         plugin_deploy_script_path,
+                    #         stack_action,
+                    #     ],
+                    #     stdout=subprocess.PIPE,
+                    #     stderr=subprocess.STDOUT
+                    # )
+
+                    # for combined_output in process.stdout:
+                    #     print(combined_output)
```

Comment:
@armab I tried this. see comment

Reply:
Thanks for trying it.

https://user-images.githubusercontent.com/1533818/198304826-491502b7-7673-46b3-99e5-ca38d64c8613.mov

I will check what's missing end to end when applying this to our script.

Reply:
this makes me think that it might be a docker or OS ([base docker image](https://github.com/bitovi/bitops/blob/main/Dockerfile#L2)) problem 🤔

Reply:
Tried editing the original Docker container inline with that version of `deploy_plugins.py` and the live output worked, outputting terraform results line-by-line as we get it.

https://user-images.githubusercontent.com/1533818/198373559-f8db6c81-fa8b-4c13-988f-f98361d26143.mov

Make sure there's no cache involved. Worst case, we can build the temporary testing Docker images later to try it again.